### ISSUE TITLE
any reason to use ui thread?

### ISSUE THREAD

**Issue by tigerbalm** [nvanbenschoten/RxSensor#4]
Is there any reason to use UI thread handler to deliver sensor event?
Is it more natural that user handle thread using subscribeOn or observeOn?

**Comment by piotr-pawlowski** [comment]
Yes, you are right. It is a huge bug, because subribeOn method is ignored in that library. All events are delivered to main thread.
